types/bootstrap/index.d.ts

```diff
@@ -39,13 +39,19 @@ declare global {
 
     interface Element {
         addEventListener(
-            type: Carousel.Events,
+            type: Carousel.Events | 'slide.bs.carousel' | 'slid.bs.carousel',
```

Comment:
If we're willing to bump the minimum TS version to 4.1, we can take advantage of [template literal types](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#template-literal-types) to simplify this and maintain a single source of truth for event names:

```suggestion
            type: Carousel.Events | `${Carousel.Events}`,
```

I couldn't find another way to access the string values of an enum without that.

Reply:
IMO Bootstrap is quite popular across different user bases, so if that can be implemented today, one should use TS versions:
https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-very-new-typescript-versions

Reply:
I don't particularly want to have to maintain two separate type definitions here, especially seeing as we can implement this without relying on newer language features. I'm inclined to ship this as-is using the duplicated literal types.